users/README.md

```diff
@@ -76,25 +76,25 @@ future changes by simply running `terraform apply
 | admin_usernames | The usernames associated with the admin accounts to be created, which are allowed to access the terraform backend and are IAM administrators.  The format first.last is recommended. | string | | yes |
 | assume_access_terraform_backend_policy_description | The description to associate with the IAM policy that allows assumption of the role with access to the Terraform backend | string | `Allow assumption of the AccessTerraformBackend role in the Terraform account.` | no |
 | assume_access_terraform_backend_policy_name | The name to assign the IAM policy that allows assumption of the role with access to the Terraform backend | string | `Terraform-AssumeAccessTerraformBackend` | no |
-| assume_iam_admin_policy_description | The description to associate with the IAM policy that allows assumption of the role to become an IAM admininistrator | string | `Allow assumption of the IamAdministrator role.` | no |
-| assume_iam_admin_policy_name | The name to assign the IAM policy that allows assumption of the role to become an IAM admininistrator | string | `AssumeIamAdministrator` | no |
+| assume_provision_account_policy_description | The description to associate with the IAM policy that allows assumption of the role to provision all AWS resources in this account | string | `Allow assumption of the ProvisionAccount role.` | no |
```

Comment:
For consistency, this should be `assume_provisionaccount_policy_description`.  See also lines 80, 84, 85 and 97.

Reply:
I did the same thing in the "shared-services" directory in #3!  I must remember incorrectly what we agreed to.